rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,421 @@
+
+Overlapping timeboxes denote concurrent work.
+
+Timeboxes with a range of days indicate that the work MAY be completed at any point 
+during the time box, but MUST be completed no later than the final day of the time box.
+
+Day 1 in the Release planning timeline begins in the third Week of the third Milestone of the current Release. For example Q4 planning begins in Q3M3W3 and continues into Q4M1. To reiterate, Planning continues into Milestone 1 of a Release as needed.
+
+* Prerequisite: Product Roadmap
+* Prerequisite: Optional Technical Objectives RFC
+* Day 1: Clarifying the Next Release
+* Day 1-15: Sketching the Release
+* Day 1-20: Creating the Engineering Plan
+
+#### Clarifying the Next Release
+[Clarifying the Next Release]:#clarifying-the-next-release
+
+***Previous Release Milestone 3 Week 3.*** The Roadmap Shepherd schedules a two-hour meeting with DCP PM and the Technical Leads to review any incomplete Objectives in the Release, the next Objectives in the DCP Roadmap, and the Technical Objectives RFC.
+
+Incomplete Objectives are the highest priority for the next Release unless emerging conditions have altered the priorities for the previous Release plan.
+
+The outcome from this meeting is the proposed set of Product and Technical Objectives for the next Release. One (and only one) Product Owner MUST be assigned to each proposed Objective. This Product Owner is responsible for refining and driving the Objective to on-time closure in coordination with Technical Leads and other collaborators.
+
+The dates for the next ZenHub Release and Milestones MUST be assigned or updated in this meeting.
+
+DCP PM MUST assign a rotating Release Shepherd to help the Product Owners and Technical Leads observe the processes and practices for the Release as outlined in this RFC. 
+
+#### Sketching the Release
+
+***Next Release Milestone 1 Week 1.*** For each Objective in the Release, its Product Owner MUST create one or more Epics in the *dcp* repo that document User Stories, Acceptance Criteria, and the Deliverable to Demonstrate.
+
+If an Objective is *well-understood*, then the Product Owner, in consultation with the relevant Technical Leads, MUST assign either a Milestone or a T-Shirt Size label for rough costing and capacity planning to the issue. Pre-defined T-Shirt Sizes are:
+
+| Size | Duration                |
+|:----:|-------------------------|
+|  XXS | 1 week                  |
+|  XS  | 2 weeks                 |
+|   S  | 4 weeks (1 X Milestone) |
+|   M  | 8 weeks (2 X Milestone) |
+|   L  | 12 weeks (1 X Release)  |
+|  XL  | 24 weeks (2 X Release)  |
+|  XXL | 36 weeks (3 X Release)  |
+
+Based on rough costing, Technical Leads and their engineering teams SHOULD provide an initial assessment for "fit" - whether any proposed Epics need to be cut or reduced in scope for the Release.
+
+It is strongly RECOMMENDED that engineering teams buffer 15% capacity for emerging issues during a Release.
```

Comment:
15% is nowhere near high enough. It will differ between teams but frankly 50% is more realistic right now.

Reply:
See the [conversation](https://github.com/HumanCellAtlas/dcp-community/pull/92#discussion_r308424078) above. I plan to change to _15% **minimum**_ in the next set of commits.